### PR TITLE
Reformat UGDG

### PR DESCRIPTION
**Pull request by Domszy**
### Description

Fix the bugs mentioned during the dry run and made several improvements to the document. Will update this as time goes along. 

**Improvements Made:**
1. I've added these in hopes to make the UG easier to read.
montero

2. Added numbering to the different sections

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Code quality improvements

### Checklist

- [ ] I have tested this code
- [ ] I have updated the documentation

### CODE REVIEW

**Comment by benedictchuajj** [comment]
@Domszy, I have updated the portion of the documentation bugs that I'm in charge of in this branch. 

Will close #203

**Review by benedictchuajj — APPROVED** [review]
LGTM! We can merge this in first if it's ready and create a separate PR for future additions~ 

Thanks @Domszy!! 😎  👍  💯 

Note: Also closes #168

**Comment by benedictchuajj** [comment]
@AY2122S1-CS2103T-T17-3/developers

Would appreciate if another person could help with a second review it as this is a big PR.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -123,236 +167,230 @@ Similar to NextMeeting, the LastMet is automatically updated from a NextMeeting
...
-Format: `delete {CLIENT'S ID}...`
-
-* Multiple clients can be deleted at the same time by their ids separated by a space.
-
-Examples:
-* `delete 7` will deletes client with client id 7
-* `delete 4 8 6` will deletes the clients whose client id is 4, 6 and 8
-
-### List all contacts : `list`
+| Format | `delete CLIENT'S ID...`|
```
Comment (wyrchris) [inline docs/UserGuide.md:203]: Perhaps we should use underscore or brackets, else it give the impression that it is two separate input
```suggestion
| Format | `delete CLIENT_ID...`|
```

**Review by wyrchris — APPROVED** [review]
LGTM! However, the quickstart UI does not seem to have clickable tags #199. Also, the quickstart delete description is not correct #181.

**Comment by Domszy** [comment]
Thanks @wyrchris really appreciate the input!

**Pull request merged**